ccp/vars

```diff
@@ -7,6 +7,12 @@ SUPPORT_EMAIL=support-ccp@dkfz-heidelberg.de
 PRIVATEKEYFILENAME=/etc/bridgehead/pki/${SITE_ID}.priv.pem
 
 BROKER_URL_FOR_PREREQ=$BROKER_URL
+DEFAULT_LANGUAGE=DE
+DEFAULT_LANGUAGE_LOWER_CASE=${DEFAULT_LANGUAGE,,}
```

Comment:
Please add those two values directly to the teiler modules compose or bash script.

Reply:
DEFAULT_LANGUAGE shoudn't be a property for the whole system?